bugsnag-plugin-react-native/src/main/java/com/bugsnag/android/BugsnagReactNativePlugin.kt

```diff
@@ -51,7 +51,7 @@ class BugsnagReactNativePlugin : Plugin {
         notifier.name = "Bugsnag React Native"
         notifier.url = "https://github.com/bugsnag/bugsnag-js"
         notifier.version = jsVersion
-        notifier.dependencies.add(Notifier()) // depend on bugsnag-android
+        client.notifier.dependencies = listOf(Notifier()) // depend on bugsnag-android
```

Comment:
Small nit: notifier == client.notifier, so I don't think this is necessary and it makes the code harder to understand why .dependencies is set differently.

```suggestion
        notifier.dependencies = listOf(Notifier()) // depend on bugsnag-android
```

Reply:
Updated as per suggestion.